msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,212 @@
+### Register your single-page application
+
+You need to create a Microsoft Azure App registration for your add-in on the Azure portal. The app registration must have a name, supported account type, and SPA redirect. If your add-in requires additional app registration beyond NAA and SSO, see [Register your single-page application.](/entra/identity-platform/scenario-spa-app-registration)
+
+### Add trusted brokers
+
+To enable nested authentication, your application must actively configure one or more specialized redirect URIs to signal to the Microsoft identity platform that it's allowed to be brokered by supported hosts. The redirect URI of the application must be of type **Single Page Application** and conform to the following scheme:
+
+```
+brk-<broker_application_id>://<your_domain>
+```
+
+Where <broker_application_id> is the app ID or alias of the broker or brokers you wish to trust and <your_domain> is the fully qualified domain name where your application is hosted. For example, **brk-multihub://contoso.com**.
+
+For Teams only apps, add two Redirect URIs to indicate you trust only Teams to broker tokens for you:
+
+* brk-5e3ce6c0-2b1f-4285-8d4b-75ee78787346://<your_domain>
+* brk-1fec8e78-bce4-4aaf-ab1b-5451cc387264://<your_domain>
+
+If your app has been upgraded to also run in Outlook and Microsoft365.com (in addition to Teams) then you just need to add one Redirect URI:
+
+```http
+brk-multihub://<your_domain>
+```
+
+Your domain should only include the origin and not sub-paths. For example:
+
+* Correct: brk-multihub://myapp.teams.microsoft.com
+* Incorrect: brk-multihub://myapp.teams.microsoft.com/go
+
+### Enable native bridging
+
+Configure your add-in to use nested app authentication by setting the `supportsNestedAppAuth` property to `true` in your MSAL configuration. This enables MSAL to use APIs on its native application host such as Outlook to acquire tokens for your application. If you don't set this property, MSAL uses the default JavaScript-based implementation to acquire tokens for your application, which might lead to unexpected auth prompts and unsatisfiable conditional access policies when running inside of a webview.
+
+```javascript
+
+// Config object to be passed to Msal on creation
+const msalConfig = {
+  auth: {
+      clientId: "<your_app_id>",
+      authority: "<https://login.microsoftonline.com/common>",
+      supportsNestedAppAuth: true // Enable native bridging.
+  }
+};
+```
+
+### Initialize public client application
+
+> [!NOTE]
+> Ensure that you initialize TeamsJS before you initialize MSAL (Microsoft Authentication Library), else Nested app authentication might fail.
+
+You need to initialize MSAL and get an instance of the public client application. This is used to get access tokens when needed. We recommended creating the public client application in the `Office.onReady` method.
+
+```javascript
+let pca = undefined;
+
+// Initialize the public client application
+Office.onReady(async (info) => {
```

Comment:
This has changed. It is no longer publicClientNext but is instead nestablePublicClient. Here is the new guidance pulled from our 1P onboarding guide. I will follow up with updated text and code samples

Reply:
Updated